go.mod

```diff
@@ -15,4 +14,9 @@ require (
 	golang.org/x/net v0.0.0-20200707034311-ab3426394381
 	golang.org/x/text v0.3.3 // indirect
 	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
+	gorm.io/driver/mysql v1.0.6
+	gorm.io/driver/postgres v1.0.8
```

Comment:
Thanks for finally adding support for postgres. We will have to add custom support for M2M and the `.database` file and maybe a few more things. I'll have to check the complete source code for any "hard coded" database type selection.